src/components/sessions/activeSession/SessionProgress.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Icon } from 'semantic-ui-react'
+import { FormattedMessage } from 'react-intl'
+
+import Question from './Question'
+
+const SessionProgress = ({ intl }) => (
```

Comment:
Because it is displaying the progress of the session! :D But i accept renaming the component. Just didn't find a better name.